src/holdingpen/containers/DashboardPage.jsx

```diff
@@ -1,10 +1,65 @@
 import React, { Component } from 'react';
+import data from '../report-errors.json';
+import CategoryCard from '../components/CategoryCard';
+import ExceptionsTable from '../components/ExceptionsTable';
+import { Row, Col } from 'antd';
 
 class DashboardPage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      exceptions: this.mapData(data),
+      collections: this.calculateExceptionsMetadata('collection')
+    }
+  }
+
+  mapData = (data) => {
```

Comment:
use `mapData(data) {...` instead. And if you don't need `this` inside then make it static or move it as a utility function outside of the class.